scripts/question_answering/README.md

```diff
@@ -184,3 +184,49 @@ For reference, we have also included the results of original version from Google
 |Google ELECTRA large      |      -/89.7     |     -/88.1   |
 |Google MobileBERT         |   90.0/82.9	|   79.2/76.2   |
 |Fairseq RoBERTa large     |   94.6/88.9    |	89.4/86.5   |
+
+### Run with AWS Batch
+We can quickly run the squad finetuning via the [AWS Batch support](../../tools/batch).
```

Comment:
is the batch support refactored so that it can be bootstrapped in *any* AWS account? including VPC setup, security group, etc?

Reply:
if it doesn't work we should avoid advertising this as user facing feature.

Reply:
I think it's for the purpose of running the experiments quickly if you have batch access. Otherwise, we will end up writing everything inside the batch folder.

Reply:
it's also ok to just keep the script somewhere local or in a gist.

Reply:
I've also considered this. The problem is that it will increase the communication time and it can add some values to the user if we will later have the cloudformation script.

Reply:
if the feature doesn't work for a user as is, it will cause confusion and friction.

Reply:
keeping it as part of infra code is also ok.

Reply:
May be let's still keep it in the batch folder.